ci-signing/build.sbt

```diff
@@ -0,0 +1,2 @@
+addSbtPlugin("com.codecommit" % "sbt-github-actions" % "0.14.2")
+addSbtPlugin("io.crashbox" % "sbt-gpg" % "0.2.1")
```

Comment:
Is Scala Steward smart enough to find per-directory builds?

Reply:
I would think so, since it can also update files in the `project/` directory. This unusual setup is necessary so that the plugin can host itself (see the symlinks in `project/`).